source/fundamentals/builders/sort.txt

```diff
@@ -15,8 +15,12 @@ Sorts Builders
 Overview
 --------
 
-In this guide, we show you how to specify **sort criteria** for your
-queries using **builders**.
+In this guide we show you how to use **builders** to specify
+**sort criteria** for your queries.
```

Comment:
nitpick: same as previous comment, I think the original was more focused on the page contents. Builders are used and taught, but the focus is on what can be done with them.